src/components/m-table-header.js

```diff
@@ -18,7 +18,7 @@ export class MTableHeader extends React.Component {
         let content = columnDef.title;
 
         if(this.props.draggable) {
```

Comment:
Maybe you can lift the logic in line 21 up if you rename to `draggable`, example: 

```
if(this.props.draggable && columnDef.draggable) {
  ...
}
```